cmd/helm/root.go

```diff
@@ -45,12 +45,12 @@ Common actions from this point include:
 
 Environment:
   $HELM_HOME          set an alternative location for Helm files. By default, these are stored in
-                      "$XDG_CONFIG_DIR/helm" ("$HOME/.config/helm" on Linux,
-                      "%APPDATA%\helm" on Windows and "$HOME/Library/Preferences" on OSX).
-                      NOTE: if there is old-style "$HOME/.helm" directory, it will be used.
+                      $XDG_CONFIG_DIR/helm ($HOME/.config/helm on Linux, %APPDATA%\helm on Windows
+                      and $HOME/Library/Preferences/helm on macOS).
+                      NOTE: if there is old-style $HOME/.helm directory, it will be used.
```

Comment:
I think this would read better like this:
```
NOTE: if $HOME/.helm exists, it will be used instead of $XDG_CONFIG_HOME/helm.
```

Reply:
Yeah, this looks better

Reply:
On the other hand, it doesn't describe the behavior as it is. The more accurate description of behavior should be like this:
```
NOTE: if default does not exist but $HOME/.helm does, the last will be used.
```

What do you think about this?

Reply:
Made a similar comment in a previous review: https://github.com/helm/helm/pull/5166#discussion_r263210600

Reply:
@bacongobbler 
That comment stills not close to how things work :confused: This one describes it more accurately in fact:
```
NOTE: if default does not exist but $HOME/.helm does, the last will be used.
```

Reply:
@cleas After reviewing the code again, I see what you're saying, and I agree, though I think it could use just a little tweak:

NOTE: if **the default directory** does not exist but $HOME/.helm does, the **latter** will be used.